installer/datafiles/base_omsagent.data

```diff
@@ -203,7 +203,8 @@ if ${{PERFORMING_UPGRADE_NOT}}; then
     do
         echo "Deleting certs and conf directories for workspace $ws_id if empty..."
         rmdir /etc/opt/microsoft/omsagent/$ws_id/certs/ 2> /dev/null
-        rmdir /etc/opt/microsoft/omsagent/$ws_id/conf/ 2> /dev/null       
+        rmdir /etc/opt/microsoft/omsagent/$ws_id/conf/ 2> /dev/null
```

Comment:
Not sure why this line was marked as changed... It doesn't seem to be changed at all.